content/en/docs/features/visibility.md

```diff
@@ -1,10 +1,20 @@
 ---
-title: API and GPU Usage Visibility
-linkTitle: API/GPU Visibility
+title: API and GPU Usage Optimization
+linkTitle: API/GPU Visibility and Optimization
```

Comment:
How about? The `linkTitle` simplifies long titles, ensuring the sidebar is easier to read in limited space.
```suggestion
linkTitle: API/GPU Usage Optimization
```

Reply:
I see. Looks good!